### ISSUE TITLE
The Hamburger menu button lacks accessible name

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3155]
## Description  
The Hamburger menu button lacks accessible name. Screen readers will announce the word "button" when reading this control. The user will not understand what the button does without more information.

## Location     
 Mobile iOS home page

## Priority     
High 

## Recommended fix      
Change to "main menu" by either using visual text, accessible offscreen text or aria-label.

/* .CSS class for accessible offscreen text  */
.offscreenText {position: absolute;clip: rect(1px 1px 1px 1px); /* IE6, IE7 */clip: rect(1px, 1px, 1px, 1px);clip-path: inset(50%);padding: 0;border: 0;height: 1px; width: 1px; overflow: hidden;white-space: nowrap;}

<button aria-label="main menu" class="header-hamburger-trigger" type="button">
<span></span>
</button>

## LOE  
Fast

**Comment by coryrylan** [comment]
fixed in https://github.com/vmware/clarity/pull/3206
